src/effects/effectsmanager.cpp

```diff
@@ -70,81 +70,106 @@ const QSet<ChannelHandleAndGroup>& EffectsManager::registeredChannels() const {
     return m_pEffectChainManager->registeredChannels();
 }
 
-const QList<QString> EffectsManager::getAvailableEffects() const {
-    QList<QString> availableEffects;
+bool alphabetizeEffectNameIdPairs(const QPair<QString, QString>& pair1,
```

Comment:
Using QPair is an antipattern. Please create your own proper named struct or class.